src/containers/HomeViewContainer.js

```diff
@@ -194,28 +219,40 @@ function HomeViewContainer(props) {
-                  <Link to="/course"><Button colorScheme="teal" variant="solid" size="lg" mr={4}>開始使用</Button></Link>
-                  <Link to="/about"><Button colorScheme="teal" variant="outline" size="lg" mr={4}>了解更多</Button></Link>
-                </Flex> 
-                <Flex w={["80vw","80vw","50vw","25vw"]} justifyContent={["center","start" ]} alignItems="start" flexDirection="column" bg="teal.200" borderRadius="xl" boxShadow="xl" p="4" mt="8">
-                  <Text fontSize="xl" fontWeight="800" color="gray.700" mb="2">👋 We are hiring!</Text>
-                  <Text fontSize="md" fontWeight="500" color="gray.600">新夥伴招募中，想跟我們一起打造更優質的選課系統嗎？ 快來加入我們吧！🥰</Text>
-                  <Flex w="100%" justifyContent="space-between" alignItems="center" flexDirection="row">
-                    <Text fontSize="sm" fontWeight="400" color="gray.500" mt="4">Team NTUCourse Neo - 20220303</Text>
-                    <Link to="/recruiting"><Button colorScheme="teal" variant="solid" size="sm" mt="4" rightIcon={<FaArrowRight />}>加入我們</Button></Link>
-                  </Flex>
+          <Flex justifyContent={["center","space-between" ]} flexDirection={{base: 'column-reverse', lg: 'row'}} alignItems="center" w="90vw">
+            <Flex flexDirection="column" pt={10}>
+              <Text align={{base: 'center', lg: 'start'}} fontSize={["4xl","6xl"]} fontWeight="800" color="gray.700">Course Schedule</Text>
+              <Text align={{base: 'center', lg: 'start'}} fontSize={["4xl","6xl"]} fontWeight="extrabold" color="gray.700" mt={-4} mb={2}>Re-imagined.</Text>
+              <Text align={{base: 'center', lg: 'start'}} fontSize={["xl","3xl"]} fontWeight="500" color="gray.500">修課安排不再是難事。</Text>
+              <Spacer my={4}/>
+              <Flex justifyContent={{base: 'center', lg: 'start'}} alignItems="center" flexDirection="row">
+                <Link to="/course"><Button colorScheme="teal" variant="solid" size="lg" mr={4}>開始使用</Button></Link>
+                <Link to="/about"><Button colorScheme="teal" variant="outline" size="lg" mr={4}>了解更多</Button></Link>
+              </Flex> 
+              <Flex alignItems="start" justifyContent={{base: 'center', lg: 'start'}}>
+                <Flex flexDirection="column" alignItems="start">
+                  <AnimatePresence initial={true} exitBeforeEnter={true}>
+                    {
+                      newsCard.map((item, index) => { // eslint-disable-line
+                        if (index === displayingCard){
+                          return (
+                            <motion.div key={index} initial={{ opacity: 0, y: -20 }} animate={{ opacity: 1, y: 0 }} exit={{ opacity: 0, y: 20 }} transition={{duration: 0.4}}>
+                              {item}
+                            </motion.div>
+                          )
+                        }
+                      })
+                    }
```

Comment:
Maybe `const item = newsCard[index]` can also achieve the same result...?
or do you have additional considerations? (like framework limitations...etc?)

Reply:
My bad. Resolved!